README.md

```diff
@@ -2,22 +2,35 @@
 
 ![franklin](https://s-media-cache-ak0.pinimg.com/236x/d9/f9/97/d9f997346e9e651f152ad98f3ffde330.jpg)
 
-**Please use [gofmt](https://golang.org/cmd/gofmt/) before commiting any code**
+**Please use [gofmt](https://golang.org/cmd/gofmt/) before committing any code**
 
-## Installation
+## Getting Started
+Install [Go](http://golang.org/doc/install.html) and set up your [GOPATH](https://golang.org/doc/code.html#GOPATH)
 
-1. Since this project is very lightweight and requires building docker images,
-   we are NOT currently run it using docker to avoid "Docker in Docker" (DinD) 
-   for the moment. This will likely change in the future as the need arises. 
-1. Install python 3.5
-1. Install [docker toolbox](https://www.docker.com/toolbox)
```

Comment:
Should I leave what we had before with just installing the docker toolbox? I feel like there is more to it.